exporter/exporterhelper/common.go

```diff
@@ -185,6 +197,10 @@ func (be *baseExporter) wrapConsumerSender(f func(consumer requestSender) reques
 	be.qrSender.consumerSender = f(be.qrSender.consumerSender)
 }
 
+func (be *baseExporter) setSignalType(signalType string) {
```

Comment:
it seems like this is always called after newBaseExporter.  Should we just make it part of that function signature?